systemd/ceph-radosgw@.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Ceph rados gateway
```

Comment:
I was just suggesting that we capitalize "RADOS" instead of spelling it with the lower-case "rados" :) It's a very small thing.